lib/TextRoles/BaseTextRole.php

```diff
@@ -47,4 +48,20 @@ public function setAliases(array $aliases): void
     {
         $this->aliases = $aliases;
     }
+
+
+    /*
+     * Does this text role have a special syntax like ``*cursive*``?
+     */
+    public function hasSpecialSyntax(): bool
+    {
+        return false;
+    }
+
+    /**
+     * @return array[string, array<SpanToken>]
```

Comment:
I can use the code suggestion feature when reviewing commit by commit apparently, but this does not look like valid syntax. Should be `array{string, list<SpanToken>}` I think.

Reply:
Mmh, the ci does not  complain, I have always used it like that and it always worked. Where do you have the other syntax from?

Reply:
Psalm and PHPStan